src/main/resources/META-INF/plugin.xml

```diff
@@ -163,6 +163,15 @@
                 <override-text place="GitMacheteToolbar"/>
             </action>
         </group>
+
+        <action id="GitMachete.OpenMacheteTabAction"
+                class="com.virtuslab.gitmachete.frontend.actions.base.OpenMacheteTabAction"
+                icon="MacheteIcons.MACHETE_FILE">
+            <!-- In future IntelliJ releases this group-id probably be `Git.MainMenu`
+                We can consider `VcsGroups` that seems to be more stable and place this action in VCS menu -->
+            <add-to-group group-id="GitRepositoryActions" anchor="last"/>
+            <keyboard-shortcut keymap="$default" first-keystroke="control alt M" />
```

Comment:
Hmmm note that `Ctrl-Alt-M` corresponds to `Extract method` in the standard settings, which is a relatively common keystroke (or at least common enough to get a few prospective users mad at us)... I think `Ctrl-Alt-Shift-M` would be safer

Reply:
Also... since the keymap here is `$default`... will it cover Mac OS as well? :thinking: or does Mac OS have a separate keymap?

Reply:
IIUC this keymap should cover all platform (but we can specify separate keymaps for `Mac OS X`, `Mac OS X 10.5+`, `Default for XWin` - X Window, `Default for KDE` and `Default for GNOME`)

Reply:
Shortcut changed to `control alt shift M`

Reply:
Ok, let's leave as is as for now, `Ctrl-Alt-Shift-M` is still a valid keystroke on Mac OS (as in, `Ctrl` is still used for many shortcuts despite `Cmd` being more prominent in general0